src/mpid/ch4/shm/posix/eager/fbox/fbox_types.h

```diff
@@ -19,12 +19,12 @@
 
 typedef struct {
 
-    volatile uint64_t data_ready;
+    volatile uint64_t data_ready;       /*  takes a cache line */
 
```

Comment:
What do you mean `data_ready` "takes a cache line"?

Reply:
It was just a comment: data_ready (which is a flag) is 64-bit int, so it takes cache line.
Both sender and receiver are writing to this flag, so it is better to occupy a while cache line.